raiden_installer/base.py

```diff
@@ -118,6 +117,11 @@ def load(cls, file_path: Path):
 
         _, _, settings_name = file_name.split("-")
 
+        try:
+            settings = load_settings(settings_name)
+        except FileNotFoundError as exc:
+            raise ValueError(f"There is no settings {settings_name} for config {file_path}")
```

Comment:
I'm missing context, but "no settings X for config Y" reads strangely for me, since in my mind settings and configuration are the same thing.

Reply:
I'll change it to `"There are no Wizard settings {settings_name} for Raiden configuration {file_path}"`. Thanks